lib/notifications/display.dart

```diff
@@ -114,8 +144,8 @@ class NotificationDisplayManager {
       channelId: NotificationChannelManager.kChannelId,
       groupKey: groupKey,
 
-      contentTitle: title,
-      contentText: data.content,
+      messagingStyle: messagingStyle,
+      number: messagingStyle.messages?.length,
```

Comment:
nit: put these after the color and icon, like in the corresponding zulip-mobile code

The icon and its color are the first thing the user sees, before unfurling the notification. So they logically come before the contents.

(That would have been clever in the old code too, oh well.)

Reply:
Also there's a TODO comment down there which this commit can delete :slightly_smiling_face: